client/src/pages/Blog.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext} from "react";
+import {UserContext}from '../contexts/UserContext'
+
+
+function Blog(props){
+  
+  const {user} = useContext(UserContext)
```

Comment:
Do you have a linter set up? Notice there is inconsistencies in terms of semicolons and spacing